source-stable.yml

```diff
@@ -13,6 +13,11 @@ version_control:
       type: archive
       url: http://downloads.sourceforge.net/project/opencvlibrary/opencv-unix/2.3.1/OpenCV-2.3.1a.tar.bz2
       update_cached_file: false
+    - external/poco:
+      type: archive
+      url: https://github.com/pocoproject/poco/archive/poco-1.6.0-release.zip
+      update_cached_file: false
+      archive_dir: poco-poco-1.6.0-release
```

Comment:
You don't need to define this twice if this is the same version. the stable is only if on "stable" a different version then on master is used.